From 41a94e0551b266e3e993c373f6b04f158848e643 Mon Sep 17 00:00:00 2001
From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Thu, 14 Jan 2010 19:33:20 +0100
Subject: [PATCH 1/3] remove prefix from search paths

configure adds prefix/include and prefix/lib to the search paths.
This breaks cross-compiling with DESTDIR
This patch removes this broken stuff.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 configure.in |   24 ------------------------
 1 files changed, 0 insertions(+), 24 deletions(-)

diff --git a/configure.in b/configure.in
index 1287d32..72ebdda 100644
--- a/configure.in
+++ b/configure.in
@@ -445,30 +445,6 @@ darwin*)
 esac
 AC_SUBST(FRAMEWORKS)
 
-#
-# If using $prefix we add "$prefix/include" to the include search path
-# and "$prefix/lib" to the library search path.
-#
-if test "x$prefix" != "x" ; then
-	AC_MSG_CHECKING(whether to use $prefix for headers and libraries)
-	if test -d $prefix/include ; then
-		AC_MSG_RESULT(yes)
-		#
-		# Arrange that we search for header files in "$prefix/include", as
-		# various packages we use may have been installed under "$prefix/include".
-		#
-		CFLAGS="$CFLAGS -I$prefix/include"
-		CPPFLAGS="$CPPFLAGS -I$prefix/include"
-
-		#
-		# Arrange that we search for libraries in "$prefix/lib".
-		#
-		AC_WIRESHARK_ADD_DASH_L(LDFLAGS, $prefix/lib)
-	else
-		AC_MSG_RESULT(no)
-	fi
-fi
-
 dnl Look in /usr/local for header files and libraries ?
 dnl XXX FIXME don't include /usr/local if it is already in the system
 dnl search path as this causes gcc 3.2 on Linux to complain about a change
-- 
1.6.6

